src/ucp/core/ucp_context.c

```diff
@@ -84,12 +89,19 @@ static ucs_config_field_t ucp_config_table[] = {
    "Estimation of buffer copy bandwidth",
    ucs_offsetof(ucp_config_t, ctx.bcopy_bw), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"ATOMIC_MODE", "device",
+   "Atomic operations synchronization mode.\n"
+   " host   - operations are atomic with respect to the host processor.\n"
```

Comment:
the terminology in verbs is "HCA/CPU"
however i prefer to have a more generic terminology in UCP